docs/DevelopmentSetup/windows_installation.rst

```diff
@@ -46,3 +56,24 @@ Once this has completed, start a command shell in your FLINT repository folder.
 
     # OR Visual Studio Solution (2017)
     cmake -G "Visual Studio 15 2017" -DCMAKE_INSTALL_PREFIX=C:/Development/Software/moja -DVCPKG_TARGET_TRIPLET=x64-windows -DENABLE_TESTS=OFF -DENABLE_MOJA.MODULES.ZIPPER=OFF -DCMAKE_TOOLCHAIN_FILE=c:\Development\moja-global\vcpkg\scripts\buildsystems\vcpkg.cmake ..
+
```

Comment:
The hard-coded paths here will only work if the development environment matches exactly. I used relative paths as follows.

```
# from ..\moja\FLINT\source\build
cmake -G "Visual Studio 16 2019" -DCMAKE_INSTALL_PREFIX=../../.. -DVCPKG_TARGET_TRIPLET=x64-windows -DENABLE_TESTS=OFF -DENABLE_MOJA.MODULES.ZIPPER=OFF -DCMAKE_TOOLCHAIN_FILE=..\..\..\vcpkg\scripts\buildsystems\vcpkg.cmake ..
```

Reply:
Updated